zenodo/modules/records/utils.py

```diff
@@ -27,13 +27,20 @@
 
 from __future__ import absolute_import, print_function
 
+from os.path import dirname, join
+
 from flask import current_app
+from invenio_db import db
+from invenio_pidstore.models import PersistentIdentifier, PIDStatus
 from invenio_records.api import Record
 from invenio_search import current_search
 from invenio_search.utils import schema_to_index
+from lxml import etree
+from sqlalchemy import or_
 from werkzeug.utils import import_string
 
 from zenodo.modules.openaire import current_openaire
+from zenodo.modules.records.httpretty_mock import httpretty
```

Comment:
Since this is importing a monkeypatch, just to be safe it would be better to do the import inside the function/task that uses it (ie. inside `xsd41`).